### PR TITLE
Fix console warnings

### PR DESCRIPTION
**Pull request by Beck-berry**
## Proposed Changes
  - removed `isRequired` labels of props which can be `undefined`
  - added unique keys to lists
  - fixed nesting issues
  - added missing translations

## Review
  - Check translations

### CODE REVIEW

File: app/component/map/popups/DynamicParkingLotsPopup.js [hunk 1]
```diff
@@ -57,7 +57,7 @@ class DynamicParkingLotsPopup extends React.Component {
       );
     }
 
-    return <div className="padding-vertical-small">{text}</div>;
```
Comment (leonardehrenfried) [inline app/component/map/popups/DynamicParkingLotsPopup.js:60]: The CSS class is necessary for the styling.

Screenshot from 2020-07-27 10-43-55
Screenshot from 2020-07-27 10-43-48

Can you find a solution for this?

Comment (Beck-berry) [reply app/component/map/popups/DynamicParkingLotsPopup.js:60]: I do not see any difference now, so please check the latest updates, maybe it fixed this (however it wasn't the main purpose..) If you still have this issue, I'll look for another solution.

Comment (leonardehrenfried) [reply app/component/map/popups/DynamicParkingLotsPopup.js:60]: It's the padding of "3 von 24 Stellplätzen geöffnet".

Comment (leonardehrenfried) [reply app/component/map/popups/DynamicParkingLotsPopup.js:60]: Screenshot from 2020-07-29 09-49-10

Just checked it and the padding is not restored.

Comment (Beck-berry) [reply app/component/map/popups/DynamicParkingLotsPopup.js:60]: Ok, I see, check now pls!

**Comment by leonardehrenfried** [comment]
Looks good, thanks!

**Pull request merged**